### PR TITLE
#91 add eclipse IDE support

### PR DESCRIPTION
**Pull request by caofangkun**
heron_eclipse

### CODE REVIEW

**Comment by kramasamy** [comment]
nice work @caofangkun. @prabhuinbarajan - It will be great if you could review this.

**Comment by kramasamy** [comment]
@prabhuinbarajan - any luck with this?

File: scripts/setup-eclipse.sh [hunk 1]
```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+#!/bin/bash
```
Comment (objmagic) [inline scripts/setup-eclipse.sh]: please remove duplicate Bash shebang...

**Comment by objmagic** [comment]
@kramasamy @caofangkun 

I tested out and this looks fine. Please remove the duplicate shebang and I will merge this.

**Pull request merged**